### DISCUSSION TITLE
How to generate CSS styles into jain format in grapejs ?

### DISCUSSION THREAD

**Discussion by senthil177cse** [GrapesJS/grapesjs#4237] [Q&A]
How to generate CSS styles into jain format in grapejs ?

**Comment by artf** [comment]
jain format?

**Reply by senthil177cse** [reply]
Sorry json format...

**Reply by artf** [reply]
Check this method and its options https://grapesjs.com/docs/api/editor.html#getcss

**Reply by senthil177cse** [reply]
I did't get clear there can u send little piece of code to convert json from css

**Comment by ebalajier** [comment]
Pass the option object while calling as in below sample:

            const opts = { json: true };
            const css = editor.getCss(opts)

**Reply by senthil177cse** [reply]
Thank for your answer @ebalajier
